apps/accounts/models/provider_request.py

```diff
@@ -99,24 +102,14 @@ class ProviderRequestLocation(models.Model):
     in which the new provider offers services.
     """
 
+    name = models.CharField(max_length=255, blank=True)
+    description = models.CharField(max_length=255, blank=True)
```

Comment:
Why is the new `description` field added to the location object? It's surprising to me because we have agreed to "name", "city" and "country" before (see https://github.com/thegreenwebfoundation/admin-portal/issues/336). What is the description of a location useful for?